Documentation/Books/Manual/SUMMARY.md

```diff
@@ -301,6 +302,7 @@
     * [AWS](Deployment/Cloud/AWS.md)
     * [Azure](Deployment/Cloud/Azure.md)
   * [Production Checklist](Deployment/ProductionChecklist.md)
+  * [Migrating from a _Single Instance_ to a _Cluster_](Deployment/MigratingSingleInstanceCluster.md)
```

Comment:
Not a fan of using italicized text in the navigation to be honest. We should either not add it here, or change everywhere in the SUMMARY.

Reply:
done